charts/app-compat/templates/deployment.yaml

```diff
@@ -103,7 +103,27 @@ spec:
         - name: {{ . }}
   {{- end }}
 {{- end}}
-{{- if $.Values.topologySpreadConstraints }}
+{{- $defaultTopologySpreadConstraints := list (dict "maxSkew" 1 "topologyKey" "kubernetes.io/hostname" "whenUnsatisfiable" "ScheduleAnyway" "autoLabelSelector" true) }}
```

Comment:
define this in _helpers.tpl that is a better place to put it

Reply:
updated it as per latest discussion